database/seeds/RaceSeeder.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+use App\Models\Core\Race;
+use App\Models\Core\Size;
+use Illuminate\Database\Seeder;
+
+class RaceSeeder extends Seeder
+{
+    public function run()
+    {
+        DB::table('races')->truncate();
+
+
+        /**
+         *
+         * Insert Player Races
+         *
+         */
+
+        Race::create([
+            'name'     => 'Dragonborn',
+            'code'     => 'dragonborn',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Dwarf',
+            'code'     => 'dwarf',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Elf',
+            'code'     => 'elf',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Gnome',
+            'code'     => 'gnome',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Half Elf',
+            'code'     => 'half-elf',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Halfling',
+            'code'     => 'halfling',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Half Orc',
+            'code'     => 'half-orc',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Human',
+            'code'     => 'human',
+            'playable' => true,
+        ]);
+
+        Race::create([
+            'name'     => 'Tiefling',
+            'code'     => 'tiefling',
+            'playable' => true,
+        ]);
+
+
+        /**
+         *
+         * Insert Monster Races
+         *
+         */
+        Race::create([
+            'name'     => 'Orc',
+            'code'     => 'orc',
+            'playable' => false,
```

Comment:
Not entirely true. While orcs are not playable in '5e SRD' (the base rules as described in the player's handbook) they are playable in the Volo's Guide to Monsters expansion ruleset.

I think we might need to associate a race to a ruleset.

Reply:
I know but I just wanted some player and non player characters in my db.

Reply:
So if we had a rule set. You would have multiple (In this example) Orc races. Where they link to different rule sets.

Reply:
I'm thinking there could be two races, one monster_orc and one playable_orc

OR

the 'playable' attribute could come from the join to the ruleset

Reply:
Mmm I just think about when we have the case where a race (Or item or spell etc) doesn't exist in a ruleset like this little fucker https://www.dndbeyond.com/marketplace/source/locathah-rising .